Makefile

```diff
@@ -16,4 +18,4 @@ clean:
 		rm -rf workspace
 run:
 		$(GOBUILD) -o $(BINARY_NAME) -v ./*.go
-		./$(BINARY_NAME)
\ No newline at end of file
+		./$(BINARY_NAME) export $(ROLENAME) --helm-chart=$(HELM_EXAMPLE) --workspace=$(WORKSPACE)
```

Comment:
I am not sure the make target "run" now truly captures what this is doing, since it is doing a "make run" of the nginx example, and not really running the executable in a "normal" mode.  We can revisit this later though, as it isn't super important for now.

Reply:
add make example